pkg/source/client.go

```diff
@@ -20,6 +20,7 @@ import (
 	"context"
 	"fmt"
 
+	"knative.dev/eventing-kafka/pkg/channel/consolidated/utils"
```

Comment:
should `utils` be in "common", eventually too ?

Reply:
yeah, I have a few TODOs around that.

Reply:
Won't do this now. I checked and there's nothing shared between the consolidated and the distributed in that package.

I moved some other stuff to a common place though.